doc/manual.rst

```diff
@@ -2679,6 +2679,11 @@ nor can their address be taken. They cannot be assigned new values.
 
 For let variables the same pragmas are available as for ordinary variables.
 
+As ``let`` statements are immutable after creation they need to define a value
+when they are declared. The only exception to this is if the ``{.importc.}``
+pragma (or any of the other ``importX`` pragmas) is applied, in this case the
+value is expected to come from native code, typically a C/C++ ``const``.
```

Comment:
you're leaving out a super important use case and in fact you should add a test for it: C macros

```nim
when defined(c) or defined(cpp):
  {.emit:"""
  const int foo1 = 123;
  #define foo2 124
  """.}
```

Reply:
Added a test case for it